### PR TITLE
Feat: add back to dataverse button on dataverse items pages

### PR DESCRIPTION
**Pull request by lolottetheclash**
This PR adds a previous button which is now displayed on `Dataset`, `Dataspace` and also `Service` page.
This button allows for now to navigate to `Dataverse` page (even if we come from `Home` page).
It completes this [issue](https://github.com/okp4/dataverse-portal/issues/97) and  [this one](https://github.com/okp4/dataverse-portal/issues/168).

image

image

### CODE REVIEW

**Review by 0xLodz — COMMENTED** [review]
@lolottetheclash  I validate your commit 1430352ed64aa2ecd14131c4920ca25693a262a9 👍 
However I can't approve the PR because it contains commits that have been rewritten (hashes changes) from another PR or are not up to date. Could you please base your work on the branch `feat/dataverse-tags`, thus we quickly see the change you made, without changing base branch commits hashes? Thanks

**Review by 0xLodz — APPROVED** [review]
Base branch is now [feat/dataverse-tags](https://github.com/okp4/dataverse-portal/tree/feat/dataverse-tags). I am wondering if we merge this one in it, or if we wait for `feat/dataverse-tags` to be marged?

File: src/ui/view/dataverse/component/pageTemplate/pageTemplate.tsx
Comment (lolottetheclash) [inline src/ui/view/dataverse/component/pageTemplate/pageTemplate.tsx:42]: To keep consistency, could you add "dataverse" to common i18n? even if translation is the same for our 3 proposed languages

File: src/ui/i18n/translations/common_de.json
Comment (ErikssonJoakim) [inline src/ui/i18n/translations/common_de.json:17]: Why not just add the text directly in the element since the translations are all the same?

File: src/ui/view/dataverse/component/pageTemplate/pageTemplate.tsx
Comment (ErikssonJoakim) [inline src/ui/view/dataverse/component/pageTemplate/pageTemplate.tsx:40]: back doesn't tell us much about the element. Seeing as this a container for the back button I'd call it that

```suggestion
        <div className="okp4-dataverse-portal-dataverse-back-button-container">
```

File: src/ui/view/dataverse/component/pageTemplate/pageTemplate.scss
Comment (ErikssonJoakim) [inline src/ui/view/dataverse/component/pageTemplate/pageTemplate.scss:26]: This style can be simplified

```suggestion
        display: flex;
        align-items: center;
        column-gap: 5px;

        .okp4-dataverse-portal-dataverse-back-button {
          all: unset;
          display: flex;
          align-items: center;
```

**Review by ErikssonJoakim — APPROVED** [review]
Looks good Just have some refactoring suggestions. To do with as you wish 😄

File: src/ui/view/dataverse/component/pageTemplate/pageTemplate.tsx
Comment (0xLodz) [reply src/ui/view/dataverse/component/pageTemplate/pageTemplate.tsx:40]: good point!

File: src/ui/view/dataverse/component/pageTemplate/pageTemplate.scss
Comment (0xLodz) [reply src/ui/view/dataverse/component/pageTemplate/pageTemplate.scss:26]: Good point to discuss about. If we define a global style for the button, unsetting all props would give us a different result. In that case, your proposal works.
We can also reset it globally in order to avoid to unset every new button. What do you think @ErikssonJoakim and @lolottetheclash ? Same for other props like `box-sizing`, etc. What are you thoughts about reset? I particularly appreciate this [modern proposal](https://www.joshwcomeau.com/css/custom-css-reset/).

File: src/ui/i18n/translations/common_de.json
Comment (ErikssonJoakim) [reply src/ui/i18n/translations/common_de.json:17]: Ah yes I see I had misunderstood the conversation, sorry. I guess it could serve a purpose if we want to add translations where 'dataverse' wouldn't work

File: src/ui/view/dataverse/component/pageTemplate/pageTemplate.tsx
Comment (ErikssonJoakim) [reply src/ui/view/dataverse/component/pageTemplate/pageTemplate.tsx:40]: Yes you're right. I would keep it short though.

File: src/ui/view/dataverse/component/pageTemplate/pageTemplate.scss
Comment (ErikssonJoakim) [reply src/ui/view/dataverse/component/pageTemplate/pageTemplate.scss:26]: Yes we could definitely reset it globally. I think it would be good if we only introduce resets when they are necessary though. 

Seeing as a global reset would implicate all buttons though I would prefer this to be done in a separate refactor PR to better scope the issue

Comment (0xLodz) [reply src/ui/view/dataverse/component/pageTemplate/pageTemplate.scss]: Sure, we don't have to add all of these resets. The goal is to avoid resetting default behaviours in many components. Will make a proposal for it, that can be handled in a different PR

File: src/ui/view/dataverse/component/pageTemplate/pageTemplate.scss
Comment (ErikssonJoakim) [inline src/ui/view/dataverse/component/pageTemplate/pageTemplate.scss:11]: The positioning of this element in the grid is weak as it implicitly takes the first column in the first row which can give unexpected side effects in the future. 

I would prefer that you give it the whole length of the row.

```suggestion
      @include grid-item($colStart: 1, $colEnd: -1, $rowStart: auto, $rowEnd: auto);
      display: flex;
```

**Pull request merged**
